### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by geralddejong**

### CODE REVIEW

File: app/plugins/CorePlugin.scala [hunk 1]
```diff
@@ -36,22 +36,22 @@ class CorePlugin(app: Application) extends CultureHubPlugin(app) {
...
-      titleKey = "ui.label.administration",
-      mainEntry = Some(MenuElement("/organizations/%s/admin".format(configuration.orgId), "ui.label.administration")),
+      titleKey = "_hub.Administration",
+      mainEntry = Some(MenuElement("/organizations/%s/admin".format(configuration.orgId), "_hub.Administration")),
       roles = Seq(Role.OWN)
     ),
     MainMenuEntry(
       key = "groups",
-      titleKey = "thing.groups",
+      titleKey = "_hubb.Groups",
```
Comment (manuelbernhardt) [inline app/plugins/CorePlugin.scala:51]: Typo `hubb`

File: app/views/Search/object.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
 <script>
     *{ Set dynamic values to use in object.js }*
     // i18n messages
-    jsLabels.relatedItems = "${messages.get('thing.relatedItems')}";
-    jsLabels.collection =  "${messages.get('thing.collection')}";
-    jsLabels.creator =  "${messages.get('thing.creator')}";
+    jsLabels.relatedItems = "${messages.get('_hubb.RelatedItems')}";
+    jsLabels.collection =  "${messages.get('_hubb.Collection')}";
+    jsLabels.creator =  "${messages.get('_hubb.Creator')}";
```
Comment (manuelbernhardt) [inline app/views/Search/object.html:13]: Typos `hubb`

File: app/views/organization/Groups/groups.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #{set 'visibility'}hidden#{/set}
 #{extends themeInfo.get('themeLayout') /}
 
-#{set title: messages.get('thing.groups') /}
+#{set title: messages.get('_hubb.Groups') /}
```
Comment (manuelbernhardt) [inline app/views/organization/Groups/groups.html:4]: Typo `hubb`

File: app/plugins/CorePlugin.scala (hunk 1 above)
Comment (geralddejong) [inline app/plugins/CorePlugin.scala:51]: That's not a typo, it's a kind of marker.

File: app/views/organization/Groups/list.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #{extends themeInfo.get('themeLayout') /}
 
-#{set title: messages.get('thing.groups') /}
+#{set title: messages.get('_hubb.Groups') /}
```
Comment (manuelbernhardt) [inline app/views/organization/Groups/list.html:3]: Typo `hubb

File: app/views/organization/Groups/list.html [hunk 5]
```diff
@@ -11,10 +11,10 @@
 <div class="row">
     <div class="groups-list span12">
         <table class="table table-striped">
-            <caption>&{'org.group.list'}</caption>
+            <caption>&{'_hub.GroupList'}</caption>
             <thead>
-                <th>&{'thing.name'}</th>
-                <th>&{'thing.action'}</th>
+                <th>&{'_hubb.Name'}</th>
+                <th>&{'_hubb.Action'}</th>
```
Comment (manuelbernhardt) [inline app/views/organization/Groups/list.html:17]: Typo `hubb`

File: app/views/tags/addRemoveTokenField.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <input type="text" id="${_id}" name="${_name}">
 #{if _removeConfirmation}
-<div id="removeConfirmation${_id}" title="&{'thing.confirmation'}" style="display: none">
+<div id="removeConfirmation${_id}" title="&{'_hubb.AreYouSure'}" style="display: none">
```
Comment (manuelbernhardt) [inline app/views/tags/addRemoveTokenField.html:3]: Typo `hubb`

File: app/views/tags/censorship.html [hunk 7]
```diff
@@ -15,7 +15,7 @@
   ~ limitations under the License.
   -->
 
-<div id="disableConfirmation" title="&{'thing.confirmation'}" style="display: none;">
+<div id="disableConfirmation" title="&{'_hubb.AreYouSure'}" style="display: none;">
```
Comment (manuelbernhardt) [inline app/views/tags/censorship.html:18]: Typo `hubb`

File: app/views/tags/displayLabels.html [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<h5>&{'thing.labels'}:</h5>
+<h5>&{'_hubb.Labels'}:</h5>
```
Comment (manuelbernhardt) [inline app/views/tags/displayLabels.html:1]: Typo `hubb`

**Pull request closed without merging**